calyx/src/passes/compile_control.rs

```diff
@@ -248,6 +265,24 @@ impl Visitor for CompileControl {
     ) -> VisResult {
         let st = &mut comp.structure;
 
+        if s.stmts.is_empty() {
```

Comment:
When would this occur? `collapse-seq` removes empty `seq`s.

Reply:
I implemented a different fix for it in this commit https://github.com/cucapra/futil/pull/190/commits/caa8fef79517f18801e0cd0aca4e28bb8177d3e3